PWGJE/PWGJE/AliAnalysisTaskJetCoreEmcal.h

```diff
@@ -0,0 +1,122 @@
+#ifndef ALIANALYSISTASKJETCOREEMCAL_H
+#define ALIANALYSISTASKJETCOREEMCAL_H
+
+//
+// Implementation of jet core task in Emcal framework
+// Basic functionality copied from AliAnalysisTaskEmcalJetSample
+//
+
+class TH1F;
+class TH1I;
+class TH2F;
+class TH3F;
+class THnSparse;
+class TRandom3;
+
+#include "AliEventCuts.h"
+#include "AliAnalysisTaskEmcalJet.h"
+#include "THistManager.h"
+
+  enum JetShapeType {
```

Comment:
This enum should be inside of your class

Reply:
Thanks, fixed